1725561714203	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1725563140620	Sync.Engine.Tabs.Tracker	DEBUG	Detected a tab change: scheduling a quick-write in 5000ms
1725563140627	Sync.Engine.Tabs.Tracker	DEBUG	Detected a tab change, but a quick-write is already scheduled
1725563140668	Sync.Engine.Tabs.Tracker	DEBUG	Detected a tab change, but a quick-write is already scheduled
1725563141267	Sync.Engine.Tabs.Tracker	DEBUG	Detected a tab change, but a quick-write is already scheduled
1725563141276	Sync.Engine.Tabs.Tracker	DEBUG	Detected a tab change, but a quick-write is already scheduled
1725563142070	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1725563142071	Sync.Service	DEBUG	User-Agent: Firefox/130.0 (Linux x86_64) FxSync/1.132.0.20240902145003.desktop
1725563142071	Sync.Service	INFO	Starting sync at 2024-09-06 00:35:42 in browser session bQPfIYPwn7Kk
1725563142071	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725563142073	Sync.Status	INFO	Resetting Status.
1725563142073	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1725563142074	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1725563142074	FirefoxAccounts	DEBUG	getOAuthToken enter
1725563142074	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725563142074	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725563142074	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1725563142078	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725563142078	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1725563142078	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/261848012/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725563142079	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1725563142079	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1725563142079	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725563142079	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1725563142080	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
